Assets/Scripts/Utilities/AudioZone.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AudioZone : MonoBehaviour 
+{
+	[SerializeField] AudioSource sourceData = null;
+	AudioSource playingClip = null;
+
+	void OnTriggerEnter( Collider col )
+	{
+		if ( col.GetComponent<PlayerActor>() )
```

Comment:
Rather than looking for `PlayerActor` specifically, we can make this more reusable by creating a tag component that indicates the AudioZone should activate.